### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by belozer**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -142,11 +144,29 @@ Guiding principles of state manager:
...
 
-- Effector is about _atomic **stores**_ - it uses stateful approach that has certain problems:
-  - probable [memory leaks](https://youtu.be/fbtElWjOXV0?t=1432)
+- Effector is about atomic **stores** — it uses stateful approach that has certain problems:
+  - probable memory leaks
+    > Like any other observable library
   - difficult [store] instance reusability (concurrence problems with SSR)
     > It can be solved, but a better way to solve it by design of library architecture and API
 - Asynchronous and probably cyclic dependencies specification
+  ```js
```
Comment (belozer) [inline docs/README.md]: Maybe need use repl?

Comment (artalar) [inline docs/README.md]: wrap it into `<details>`

File: docs/README.md [hunk 2]
```diff
@@ -142,11 +144,34 @@ Guiding principles of state manager:
 ### Why not Effector
 [link to repository](https://github.com/zerobias/effector)
 
-- Effector is about _atomic **stores**_ - it uses stateful approach that has certain problems:
-  - probable [memory leaks](https://youtu.be/fbtElWjOXV0?t=1432)
+- Effector is about atomic **stores** — it uses stateful approach that has certain problems:
+  - probable memory leaks
+    > Like any other observable library
```
Comment (antonk52) [inline docs/README.md]:
```suggestion
    > Like any other observable libraries
```

Comment (belozer) [reply docs/README.md]: @antonk52 thanks :+1:

**Pull request merged**